pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -23,7 +23,6 @@ spec:
     - exec:
         collectorName: kotsadm-rqlite-db
         name: kots/admin_console
-        containerName: rqlite
```

Comment:
The rqlite pod will always just have one container, right? That's why it's safe to assume that `rqlite` will be the container in which the command will be ran?